docs/usage/keycloak.md

```diff
@@ -0,0 +1,37 @@
+# Usage
+
+The following example shows the most minimal valid manifest to create a new Argo CD cluster with keycloak as Single sign-on provider.
+
```

Comment:
please add a note that currently only openshift is supported for keycloak provider.

Reply:
added